File: CodeTest/Python/BaekJoon/Class2_9_1259.py
# 팰린드롬

import sys


# 1. 반복문 이용해서 찾기
while True:   # 입력값이 0이 들어오면 반복문 종료
    input_num = str(sys.stdin.readline().rstrip())
    answer = ''

    # 입력값이 0이면 종료
    if input_num == '0': break

    # 글자를 하나씩 가져와서 앞글자에 붙임
    for c in input_num: answer = c + answer
        
    if input_num == answer: print('yes')
    else: print('no')

File: CodeTest/Java/BaekJoon/Class2_9_1259.java
import java.util.Scanner;

public class Class2_9_1259 {
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        String input_num = null;        
        String answer = null;

        while (true) {
            input_num = sc.next();
            answer = "";

            if(input_num.equals("0")) {break;}

            for(int i=0; i<input_num.length(); i++){
                answer = input_num.charAt(i) + answer;
            }

            if(input_num.equals(answer)) {System.out.println("yes");}
            else {System.out.println("no");}
        }
    }
}